test/Tests/src/System/File_Spec.enso

```diff
@@ -670,6 +670,27 @@ spec =
             r2.should_fail_with File_Error
             r2.catch.should_be_a File_Error.Not_Found
 
+        Test.specify "Should handle parent-less file paths" <|
+            transient = enso_project.data / "transient"
+            f1 = transient / "./test1.txt"
+            f2 = File.new ("parentlesstest" + Date_Time.now.to_unix_epoch_milliseconds.to_text + ".txt")
```

Comment:
This is creating a new file in the Current Working Directory. Obviously we need this for the test as that's the specific scenario we check.

But I think we need to be more careful with the cleanup as we should try hard to not pollute the project root by accident. Currently, if the test fails, the cleanup will not run.

I suggest to use `Panic.with_finalizer` (essentially Enso's `finally` construct) to run the cleanup regardless of test success.

Reply:
Done